### PR TITLE
Tile diversity

### PR DESCRIPTION
**Pull request by chris-addison**
We can now have a bit of tile diversity for graphical

### CODE REVIEW

File: Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs [hunk 1]
```diff
@@ -37,6 +37,24 @@ static float GetAlphaForStaticTile(Tile tile)
             return color;
         }
         
+        static Color GetColorForDoor(Tile tile)
```
Comment (mdiluz) [inline Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs:40]: Not quite sure I understand why this needs to be used intead of `GetAlphaForStaticTile`, ie. why do doors specifically want to be grey RGB not alpha?

Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs:40]: They look transparent as they're on top of a tile

Comment (chris-addison) [reply Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs:40]: image
Example

Comment (mdiluz) [reply Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs:40]: Ahh, touche

**Review by mdiluz — APPROVED** [review]

**Review by ObsessiveNerd — APPROVED** [review]

**Pull request merged**